forge-gui/res/cardsfolder/upcoming/fishing_pole.txt

```diff
@@ -0,0 +1,12 @@
+Name:Fishing Pole
+ManaCost:1
+Types:Artifact Equipment
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddAbility$ FishingPoleBaiting | Description$ Equipped creature has "{1}, {T}, Tap Fishing Pole: Put a bait counter on Fishing Pole."
+SVar:FishingPoleBaiting:AB$ PutCounter | Cost$ 1 T tapXType<1/OriginalHost/Fishing Pole> | CounterType$ BAIT | CounterNum$ 1 | Defined$ OriginalHost | SpellDescription$ Put a bait counter on ORIGINALHOST.

```

Comment:
was this tested?

Reply:
No, I'm not sure if tapXType allows this, since this is the first Equipment with an ability like this. Might need support.